src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -144,7 +148,17 @@ public override void OnEventWritten(string name, object payload)
             // We are relying here on ASP.NET Core to set duration before writing the stop event.
             // https://github.com/dotnet/aspnetcore/blob/d6fa351048617ae1c8b47493ba1abbe94c3a24cf/src/Hosting/Hosting/src/Internal/HostingApplicationDiagnostics.cs#L449
             // TODO: Follow up with .NET team if we can continue to rely on this behavior.
-            this.httpServerDuration.Record(Activity.Current.Duration.TotalMilliseconds, tags);
+            if (this.emitNewAttributes)
+            {
+                // TODO: This needs to be changed to TotalSeconds. This is blocked until we can change the default histogram.
+                // See: https://github.com/open-telemetry/opentelemetry-dotnet/issues/4797
+                this.httpServerRequestDuration.Record(Activity.Current.Duration.TotalMilliseconds, tags);
```

Comment:
same tags should not be used for both old and new.

Reply:
good catch. updated.